internal/querycoordv2/balance/balance.go

```diff
@@ -67,7 +69,18 @@ type RoundRobinBalancer struct {
 	nodeManager *session.NodeManager
 }
 
-func (b *RoundRobinBalancer) AssignSegment(collectionID int64, segments []*meta.Segment, nodes []int64) []SegmentAssignPlan {
+func (b *RoundRobinBalancer) AssignSegment(collectionID int64, segments []*meta.Segment, nodes []int64, manualBalance bool) []SegmentAssignPlan {
+	// skip out suspend node and stopping node during assignment, but skip this check for manual balance
+	if !manualBalance {
+		nodes = lo.Filter(nodes, func(node int64, _ int) bool {
+			info := b.nodeManager.Get(node)
+			if info != nil && info.GetState() == session.NodeStateNormal {
+				return true
+			}
+			return false
```

Comment:
we can simplify it with
`return info != nil && info.GetState() == session.NodeStateNormal`